core/store/migrate/migrate.go

```diff
@@ -88,7 +87,7 @@ func ensureMigrated(db *sql.DB, lggr logger.Logger) {
 				}
 			}
 
-			if _, err = db.Exec(sql, id); err != nil {
+			if _, err = db.Exec(`INSERT INTO ? (version_id, is_applied) VALUES ($1, true)`, goose.TableName()); err != nil {
```

Comment:
@jmank88 wanted to call this out specifically for your to check

Reply:
Oh, I see an issue, fixing.

Reply:
I think you might need this:
```suggestion
			if _, err = db.Exec(`INSERT INTO $1 (version_id, is_applied) VALUES ($2, true)`, goose.TableName(), id); err != nil {
```
But it may not actually work? This could have been an intentional work around that we need to comment //nolint :shrug:

Reply:
Ah, you're right. Sorry I made a separate change in 4e4f2adfae6d18269bc155cd0b8f79cf12e7673b before I saw your suggestion. This is what I did though. Should we `//nolint` instead or what makes you think it won't work?